1566558511830-SeedCategory.ts

```diff
@@ -0,0 +1,11 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class SeedCategory1566558511830 implements MigrationInterface {
```

Comment:
Is this file used? :D

Reply:
Yes. You can't even change that awful name :D